.github/workflows/github-release.yaml

```diff
@@ -0,0 +1,22 @@
+name: github-release
+
+on:
+  push:
+    tags:
+      - "[0-9]+.[0-9]+.[0-9]+"
+
+jobs:
+  github-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+
+      - name: Release to GitHub
+        run: |
+          gh release create "${{ github.ref_name }}" \
+            --draft \
+            --title "Release ${{ github.ref_name }}" \
+            --notes ""
```

Comment:
You can customize this `notes`.
My recommendation is, as commented in https://github.com/platisd/clang-tidy-pr-comments/issues/17#issuecomment-1037857937, to follow Conventional Commits and automatically generate changelogs by `git-cliff`.

Please see the followings:
- https://github.com/autowarefoundation/autoware-github-actions/pull/38#issuecomment-1030910778
- https://github.com/kenji-miyake/test-git-cliff-release/releases

How would you like?